drivers/sensor/lps22hh/lps22hh_spi.c

```diff
@@ -0,0 +1,136 @@
+/* ST Microelectronics LPS22HH pressure and temperature sensor
+ *
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Datasheet:
+ * https://www.st.com/resource/en/datasheet/lps22hh.pdf
+ */
+
+
+#include <string.h>
+#include "lps22hh.h"
+#include <logging/log.h>
+
+#ifdef DT_ST_LPS22HH_BUS_SPI
+
+#define LPS22HH_SPI_READ		(1 << 7)
+
+#define LOG_LEVEL CONFIG_SENSOR_LOG_LEVEL
+LOG_MODULE_DECLARE(LPS22HH);
+
+static struct spi_config lps22hh_spi_conf = {
```

Comment:
This should be in `struct lps22hh_config`

Reply:
moved in config